phpunit.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite name="CallFire-PHP-SDK test">
+            <directory>test</directory>
+        </testsuite>
+    </testsuites>
+    <php>
+        <env name="API_Username" value="login" />
```

Comment:
We should leave these credentials empty by default, so the integration tests don't run.